### ISSUE TITLE
Not all devices are exposed/found by homebridge

### ISSUE THREAD

**Issue by kylerw** [home-assistant/homebridge-homeassistant#70]
Cover devices will show up some times and then randomly disappear. Removing and re-adding homebridge doesn't work with adding it back.

**Comment by robbiet480** [comment]
@kylerw It's hard to know if this is a Homebridge issues, a Homebridge-homeassistant issue or a HomeKit issue. Usually it's a HomeKit problem, as it seems pretty buggy still (or at least the integration with Homebridge is).

Can you start Homebridge in debub and insecure mode? This will allow you to inspect the accessories yourself by running `curl 127.0.0.1:51826/accessories`. Make sure you do so at startup and as soon as you notice things missing to compare the two. If you aren't seeing any accessories missing from `/accessories` but missing from HomeKit you should try wiping out the `persist` and `accessories` directories. This will remove all your rooms and custom configurations in HomeKit though.

**Comment by hdemel** [comment]
I'm noticing the same issue. How would one go about starting home bridge in debug and insecure mode?

**Comment by kolossboss** [comment]
I notice a similar issue.

I added a Zwave switch for my Rollershutter. I see this Cover in my Homekit Apps just fine. 

Then I tap the button to close the Shutter and everthing seems correct, the Shutter goes down. But exact in the moment, homebridge crashed (and restarted with systemd service).
And homebridge crashed too, if I switched the button in HA. It looks like, homebridge crashed, if the status for the cover changed.

I hope this helps

I see this in the log

```
[11/19/2016, 11:12:22 AM] [HomeAssistant] Setting the state of the Jalousie_HWR to 0
[11/19/2016, 11:12:22 AM] [HomeAssistant] Successfully set position of Jalousie_HWR to 0
/usr/local/lib/node_modules/homebridge-homeassistant/accessories/cover.js:32
        .setValue(coverState, null, "internal");
        ^

TypeError: Cannot read property 'setValue' of undefined
    at Object.HomeAssistantCover.onEvent (/usr/local/lib/node_modules/homebridge-homeassistant/accessories/cover.js:32:9)
    at Object.<anonymous> (/usr/local/lib/node_modules/homebridge-homeassistant/index.js:61:19)
    at emitOne (events.js:77:13)
    at EventSource.emit (events.js:169:7)
    at _emit (/usr/local/lib/node_modules/homebridge-homeassistant/node_modules/eventsource/lib/eventsource.js:181:17)
    at parseEventStreamLine (/usr/local/lib/node_modules/homebridge-homeassistant/node_modules/eventsource/lib/eventsource.js:196:9)
    at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/homebridge-homeassistant/node_modules/eventsource/lib/eventsource.js:159:11)
    at emitOne (events.js:77:13)
    at IncomingMessage.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:146:16)
```

**Comment by landerss0n** [comment]
kolossboss, im having the same issue as you.

**Comment by robbiet480** [comment]
Closed via #73.
